tsconfig.json

```diff
@@ -22,7 +22,6 @@
         "incremental": true, // Enable incremental compilation by reading/writing information from prior compilations to a file on disk
         "removeComments": true, // Disable emitting comments
         "skipDefaultLibCheck": true, // Skip type checking .d.ts files that are included with TypeScript
-        "noImplicitReturns": true,
```

Comment:
This line from the global ts config is removed to avoid `not all code paths return a value` error that I had in my current component and to avoid further issues in the future too.